.gitignore

```diff
@@ -12,4 +12,7 @@
 *.out
 
 #.vscode configuration files
-.vscode/*
\ No newline at end of file
+.vscode/*
+
+#Goland configuration files
+.idea/*
```

Comment:
this is not `golang configuration files` but `intellij config files`

Reply:
Corrected with the new gitignore file generated from gitignore.io as suggested